src/App.vue

```diff
@@ -11,11 +12,33 @@
         <p>We're sorry, we're not able to retrieve this information at the moment, please try back later</p>
       </section>
 
-      <section v-else>
+      <section class="grid" v-else>
         <div v-if="loading">Loading...</div>
 
+        <div class="beer-filter">
+          <div class="beer-filter__item">
+            <label class="beer-filter__label">Search</label>
+            <input
+              class="beer-filter__input"
+              type="search"
+              placeholder="e.g. Punk IPA"
+              v-model="beerSearch"
+            />
+          </div>
+
+          <div class="beer-filter__item">
+            <label class="beer-filter__label">Filter by ABV</label>
+            <select class="beer-filter__input beer-filter__input--select" v-model="beerStrength">
+              <option value="all">All Beers</option>
+              <option value="weak">Weak (0 - 4.5%)</option>
+              <option value="medium">Medium (4.5% - 7.5%)</option>
+              <option value="strong">Strong (7.5% +)</option>
+            </select>
+          </div>
+        </div>
+
         <div class="beer-list" v-if="beers">
-          <Beer v-for="beer in beers" :beer="beer" :key="beer.id" />
+          <Beer v-for="beer in filterBeers" :beer="beer" :key="beer.id" />
```

Comment:
What if `filterBeers` is an empty array? It would be nice to have an empty state